src/util/constants.ts

```diff
@@ -20,127 +20,157 @@ export const baseEndpoints = {
 
   // Application Endpoints
   COMMANDS: (botID: string) =>
     `${baseEndpoints.BASE_URL}/applications/${botID}/commands`,
-  COMMANDS_GUILD: (botID: string, id: string) =>
-    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${id}/commands`,
-  COMMANDS_ID: (botID: string, id: string) =>
-    `${baseEndpoints.BASE_URL}/applications/${botID}/commands/${id}`,
-  COMMANDS_GUILD_ID: (botID: string, id: string, guildID: string) =>
-    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${guildID}/commands/${id}`,
+  COMMANDS_GUILD: (botID: string, guildID: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${guildID}/commands`,
+  COMMANDS_ID: (botID: string, commandID: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${botID}/commands/${commandID}`,
+  COMMANDS_GUILD_ID: (botID: string, commandID: string, guildID: string) =>
+    `${baseEndpoints.BASE_URL}/applications/${botID}/guilds/${guildID}/commands/${commandID}`,
 
   // Interaction Endpoints
-  INTERACTION_ID_TOKEN: (id: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/interactions/${id}/${token}/callback`,
-  INTERACTION_ORIGINAL_ID_TOKEN: (id: string, token: string) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${id}/${token}/messages/@original`,
+  INTERACTION_ID_TOKEN: (interactionID: string, token: string) =>
+    `${baseEndpoints.BASE_URL}/interactions/${interactionID}/${token}/callback`,
+  INTERACTION_ORIGINAL_ID_TOKEN: (interactionID: string, token: string) =>
+    `${baseEndpoints.BASE_URL}/webhooks/${interactionID}/${token}/messages/@original`,
   INTERACTION_ID_TOKEN_MESSAGEID: (
-    id: string,
+    applicationID: string,
     token: string,
     messageID: string,
   ) =>
-    `${baseEndpoints.BASE_URL}/webhooks/${id}/${token}/messages/${messageID}`,
+    `${baseEndpoints.BASE_URL}/webhooks/${applicationID}/${token}/messages/${messageID}`,
 
   // User endpoints
-  USER: (id: string) => `${baseEndpoints.BASE_URL}/users/${id}`,
+  USER: (userID: string) => `${baseEndpoints.BASE_URL}/users/${userID}`,
   USER_BOT: `${baseEndpoints.BASE_URL}/users/@me`,
-  USER_AVATAR: (id: string, icon: string) =>
-    `${baseEndpoints.CDN_URL}/avatars/${id}/${icon}`,
+  USER_GUILDS: `${baseEndpoints.BASE_URL}/@me/guilds`,
+  USER_AVATAR: (userID: string, icon: string) =>
+    `${baseEndpoints.CDN_URL}/avatars/${userID}/${icon}`,
   USER_DEFAULT_AVATAR: (icon: number) =>
     `${baseEndpoints.CDN_URL}/embed/avatars/${icon}.png`,
-  USER_CREATE_DM: `${baseEndpoints.BASE_URL}/users/@me/channels`,
+  USER_DM: `${baseEndpoints.BASE_URL}/users/@me/channels`,
+  USER_CONNECTIONS: `${baseEndpoints.BASE_URL}/users/@me/connections`,
+  USER_NICK: (guildID: string) => `${GUILDS_BASE(guildID)}/members/@me/nick`,
+
+  // Voice endpoints
+  VOICE_REGIONS: `${baseEndpoints.BASE_URL}/voice/regions`,
+
+  // Oauth2 endpoints
+  OAUTH2_APPLICATION: `${baseEndpoints.BASE_URL}/oauth2/applications/@me`,
```

Comment:
Is this used anywhere?

Reply:
Yes, it returns the bots Application Object:
https://discord.com/developers/docs/topics/oauth2#application-object

Reply:
Are we using it in Discordeno? That is what I meant.

Reply:
No, but I plan to add this. So devs who want to get this info can use it.

Reply:
@itohatweb when you do, please add the endpoint in THAT PR :)

Reply:
I am working on many additions of functions. Adding endpoints each time just add conflicts so I wanted to prevent this with this PR.